chore: Add Svelte 5 peer dependency

ieedan opened the pull request:
Add Svelte 5 as a peer dependency to suppress warnings from package managers.

I don't think there is any reason that this would have issues with Svelte 5. I am using it and I didn't have any issues (outside of the install warning)

ieedan committed:
Add svelte 5 peer dep

SaschaLucius commented:
thank you @ieedan. will create a release in the next days

ieedan commented:
> thank you @ieedan. will create a release in the next days

Hey thanks for merging! Any idea when you'll have the chance to create a release?

SaschaLucius commented:
@ieedan I have pushed version 0.0.4.
Have used the moment and updated all dependencies, fixed some tests and TS definitions.
If possible please give me a short ping if it now works for you as expected.

ieedan commented:
> @ieedan I have pushed version 0.0.4. Have used the moment and updated all dependencies, fixed some tests and TS definitions. If possible please give me a short ping if it now works for you as expected.

Yeah the warning is gone LGTM! Thanks a ton!
